sql/schema.sql

```diff
@@ -0,0 +1,89 @@
+-- To execute this file from SQL REPL:
+-- \i sql/schema.sql
+
+-----------------
+-- BASE TABLES --
+-----------------
+
+CREATE TABLE IF NOT EXISTS repos
+( id               INT  NOT NULL
+, repo_name        TEXT NOT NULL
+, repo_desc        TEXT NOT NULL
+, repo_url         TEXT NOT NULL -- The repo html_url
+, open_issue_count INT  NOT NULL
+, fork_count       INT  NOT NULL
```

Comment:
We probably won't need everything from here, but there is no harm in having more fields, the schema can be always adjusted later.

Reply:
Got it.